### ISSUE TITLE
Qualitative consequences of repeated eigenvalues

### ISSUE THREAD

**Issue by neldredge-unco** [twjudson/odeproject-public#7]
https://github.com/twjudson/odeproject-public/blob/43d1bb9e45837a51a1ed7d127c528ca2c1be09fb/source/linear05.ptx#L278

I think this passage could use some more explanation.  We describe the equilibrium at 0 as a "nodal source" (or sink), but what do we mean by that?  In what ways is it like a node, and in what ways is it like a source?  As it's still the case that all solutions diverge from the origin, why is it necessary to distinguish this case from an ordinary source?

Basically, this chapter discusses the algebraic issues around solving the system in the case of a repeated eigenvalue, and how the solution takes a special algebraic form.  But I think it would help to have some discussion of how the behavior of such a system differs *qualitatively* from other cases.

**Comment by twjudson** [comment]
I see your point.  We really don’t have “nodal” sources or sinks.  Actually, the repeated eigenvalues case is the situation where we are moving from spiral equilibrium solutions to nodal equilibrium solutions, but I am not quite sure what to say.  For the time being, I have eliminated the word “nodal.”  I will have to think about what, if any, elaboration is needed.



On Nov 20, 2023, at 8:36 PM, Nate Eldredge ***@***.***> wrote:


This Message Is From an External Sender
This message came from outside your organization.


https://github.com/twjudson/odeproject-public/blob/43d1bb9e45837a51a1ed7d127c528ca2c1be09fb/source/linear05.ptx#L278<https://urldefense.com/v3/__https://github.com/twjudson/odeproject-public/blob/43d1bb9e45837a51a1ed7d127c528ca2c1be09fb/source/linear05.ptx*L278__;Iw!!FvPYdGY!e7e29ZjuaHQaSmbv-fKSU4HVcQ6VznrqBlwv7DLER-gYsGgKppZYVMzU7JiRhqGU5nq3eU1LkLAdFNatiYFp9hDBwQ$>

I think this passage could use some more explanation. We describe the equilibrium at 0 as a "nodal source" (or sink), but what do we mean by that? In what ways is it like a node, and in what ways is it like a source? As it's still the case that all solutions diverge from the origin, why is it necessary to distinguish this case from an ordinary source?

Basically, this chapter discusses the algebraic issues around solving the system in the case of a repeated eigenvalue, and how the solution takes a special algebraic form. But I think it would help to have some discussion of how the behavior of such a system differs qualitatively from other cases.
